src/features/prompts/components/prompt-history.tsx

```diff
@@ -0,0 +1,75 @@
+import { StatusBadge } from "@/src/components/layouts/status-badge";
+import { type NextRouter, useRouter } from "next/router";
+
+type Prompt = {
```

Comment:
Why do we need a new type here? We already have a prompt type or not?

Reply:
Because of the join with the users table.

the .allVersions route returns the Prompt Type, we already have, plus creator: string | null according to TS.
I am curios if there is a more native t3 way to handle this.